doc/source/whatsnew/v1.1.0.rst

```diff
@@ -144,7 +144,7 @@ Reshaping
 - Bug in :meth:`DataFrame.pivot_table` when only MultiIndexed columns is set (:issue:`17038`)
 - Fix incorrect error message in :meth:`DataFrame.pivot` when ``columns`` is set to ``None``. (:issue:`30924`)
 - Bug in :func:`crosstab` when inputs are two Series and have tuple names, the output will keep dummy MultiIndex as columns. (:issue:`18321`)
-
+- Fix bug in combine.Series (:issue:`31142`)
```

Comment:
Should this be
```
:meth:`Series.combine`
```
?

Reply:
yes it should

Reply:
it should also have a few more words describing what the bug is

Reply:
looks like you already got to this, never mind